src/window_internal_glutin.rs

```diff
@@ -153,8 +153,8 @@ impl<UserEventType> WindowHelperGlutin<UserEventType>
                     err
                 )
             })?;
-
-        match self.window_context.window().set_cursor_grab(grabbed) {
+        let grabbed_enum = if grabbed { CursorGrabMode::None } else { CursorGrabMode::Confined };
```

Comment:
This part feels ugly, I've just forced a bool into one of two enum values. But there's 3 enum values now. 
`CursorGrabMode::None`
`CursorGrabMode::Confined`
`CursorGrabMode::Locked`

Reply:
Thanks for flagging this up. It looks like `Locked` can be used on Mac, and `Confined` can be used on Windows/X11. I guess we need to check the platform here and choose between them.

Reply:
Docs here: https://docs.rs/glutin/latest/glutin/window/enum.CursorGrabMode.html#variant.Confined

Reply:
Ok I'll replace that if statement with a platform check later tonight. Loving this library by the way!

Reply:
Ok so when `grabbed` is true, it will translate to `Confined` for windows or linux, `Locked` for everything else.